include/roboteam_ai/control/positionControl/pathPlanning/Voronoi.h

```diff
@@ -0,0 +1,1607 @@
+// Copyright (c) 2015-2019 Mathias Westerdahl
```

Comment:
This file should not go with this PR but rather into roboteam_utils as it's a library. Or you actually link it as a header library. Is there a way to apt-get this?

Reply:
It's a header library. No apt-get, but I attached the github link

Reply:
This can be put in the docker image.

Reply:
Done, just `#include <jc_voronoi.h>` now